react/wbooks/src/app/screens/Home/components/Book/index.js

```diff
@@ -0,0 +1,26 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+import logoBook from '../../../../assets/book-cover.png';
+
+import styles from './styles.module.scss';
+
+function Book({ title, author }) {
+  return useMemo(
+    () => (
+      <div className={styles.bookCover}>
+        <img className={styles.bookImage} src={logoBook} alt="Wolox" />
+        <h3 className={styles.title}>{title}</h3>
+        <p className={styles.author}>{author}</p>
+      </div>
+    ),
+    [author, title]
```

Comment:
Hey, don't use useMemo for renders. React already renders this only when title and author change. The useMemo hook should be use for complicated calculations before a render